src/app/home/pages/PlanDebugPage/components/TreeActionsDropdown.tsx

```diff
@@ -1,34 +1,42 @@
 import React, { useState } from 'react';
-import { Dropdown, KebabToggle, DropdownItem, Tooltip } from '@patternfly/react-core';
+import { connect } from 'react-redux';
+import { Dropdown, KebabToggle, DropdownItem, Tooltip, Alert } from '@patternfly/react-core';
 import spacing from '@patternfly/react-styles/css/utilities/Spacing/spacing';
 import { IDebugTreeNode } from '../../../../debug/duck/types';
-import { getFullKindName } from '../helpers';
+import { getOCCommandAndClusterType } from '../helpers';
+import { AlertActions } from '../../../../common/duck/actions';
 
 interface ITreeActionsDropdownProps {
   rawNode: IDebugTreeNode;
   viewRawDebugObject: (node: IDebugTreeNode) => void;
+  copiedToClipboard: (text: string) => void;
 }
 
 const TreeActionsDropdown: React.FunctionComponent<ITreeActionsDropdownProps> = ({
   rawNode,
   viewRawDebugObject,
+  copiedToClipboard,
 }: ITreeActionsDropdownProps) => {
   const [kebabIsOpen, setKebabIsOpen] = useState(false);
+  const [clusterType, setClusterType] = useState('');
 
   const onCopy = (event: React.MouseEvent<HTMLAnchorElement>) => {
     event.preventDefault();
     const clipboard = event.currentTarget.parentElement;
     const el = document.createElement('textarea');
-    el.value = `oc get ${getFullKindName(rawNode.kind)} -n ${rawNode.namespace} ${rawNode.name}`;
+    const { ocCommand, clusterType } = getOCCommandAndClusterType(rawNode);
+    setClusterType(clusterType);
+    el.value = ocCommand;
     clipboard.appendChild(el);
     el.select();
     document.execCommand('copy');
     clipboard.removeChild(el);
+    copiedToClipboard(
+      `Copied to clipboard! Run the oc get command on 
```

Comment:
@eriknelson 
Suggested rewrite of error mesage: 
"Command copied to clipboard. Run 'oc get <resource>' on the ${clusterType} cluster to view resource details."